docs/run/agent/installation.md

```diff
@@ -1,4 +1,19 @@
-# Fleet Installation
+# Installation
+To get started with the agent, create a new cluster in the Insights UI, e.g. `staging`
+
+<div class="mini-img">
+  <img :src="$withBase('/img/new-cluster.png')" alt="new cluster">
+</div>
+
+## Install the Agent
+
+When you create the cluster, you'll be prompted to install the [Insights Agent](https://insights.docs.fairwinds.com/installation/insights-agent/)
```

Comment:
is this link not going to change? `run/agent/installation`?

Reply:
Good catch! not sure why it was a full URL